lighthouse-core/test/audits/seo/font-size-test.js

```diff
@@ -234,4 +237,54 @@ describe('SEO: Font size audit', () => {
     expect(auditResult.score).toBe(1);
     expect(auditResult.notApplicable).toBe(true);
   });
+
+  describe('attributes source location', () => {
+    function runFontSizeAuditWithSingleFailingStyle(style, nodeProperties) {
+      const artifacts = {
+        URL: {finalUrl: 'http://www.example.com'},
+        MetaElements: makeMetaElements(validViewport),
+        FontSize: {
+          analyzedFailingNodesData: [
+            {textLength: 1, fontSize: 1, node: {nodeId: 1, ...nodeProperties}, cssRule: style},
+          ],
+        },
+        TestedAsMobileDevice: true,
+      };
+      return FontSizeAudit.audit(artifacts, getFakeContext());
+    }
+
+    it('to inline node stylesheet', async () => {
+      const auditResult = await runFontSizeAuditWithSingleFailingStyle({
+        type: 'Inline',
+      }, {
+        parentNode: {attributes: ['id', 'my-parent']},
+        localName: 'p',
+        attributes: ['class', 'my-p'],
+      });
+
+      assert.equal(auditResult.details.items.length, 1);
```

Comment:
expect().toHaveLength(1) for the better error message? :D also now that I twisted your arm for the others it feels weird right next to assert 😆